server/routes/handlers/assess/getDossier.ts

```diff
@@ -0,0 +1,21 @@
+import type { RequestHandler } from 'express'
```

Comment:
This looks almost. identical. to the existing getRevocationOrder handler; could just share. it and inject. the. different. bits, same. as with. the view handler? It will be half the amount of code and tests to maintain.

Reply:
Yes I thought that, but left it initially, wasn't sure if we wanted to couple the handlers or not.  I can do this though